web/src/app/modules/overview/components/tabs/tabs.component.ts

```diff
@@ -61,9 +72,30 @@ export class TabsComponent implements OnChanges, OnInit {
         this.activeTab = this.tabs[0].accessor;
         this.setMarker(this.activeTab);
       }
+
+      if (this.extView && this.tabs.length > 0) {
+        this.sliderService.activeTab.subscribe(index => {
+          this.activeTabIndex = index;
+        });
+
+        // Case where new tab is added or initial load
+        if (views.length > preViews.length || this.activeTabIndex === null) {
```

Comment:
I think this is the problem because it cannot distinguish between closing a tab vs opening one which messes up the index